packages/notifications/src/Http/Livewire/Notifications.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Filament\Notifications\Http\Livewire;
+
+use Filament\Notifications\Collection;
+use Filament\Notifications\Notification;
+use Illuminate\Contracts\View\View;
+use Livewire\Component;
+
+class Notifications extends Component
+{
+    public Collection $notifications;
+
+    protected $listeners = [
+        'notificationSent' => 'add',
+        'notificationClosed' => 'close',
+    ];
+
+    public function mount(): void
+    {
+        $this->notifications = new Collection();
+    }
+
+    public function add(array $notification): void
+    {
+        $this->notifications->push(Notification::fromLivewire($notification));
+    }
+
+    public function close(array $notification): void
+    {
+        $notification = Notification::fromLivewire($notification);
+
+        $this->notifications->map(
```

Comment:
I don't think a map is appropriate here, we can probably just use a `foreach`

Reply:
Hmm yeah, I realize this is kind of leftover dead code. We never retrieve the notification closed state, since it's set in JS based on the Livewire event. We probably do need this however, since I still have to implement notifications after redirects.

Will update for now, but great chance it'll change anyway.

Reply:
31e7fd8f5ce05daf52e94a3e59c6107f8216e08c